WordPressAuthenticator/Extensions/UIView+Helpers.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import UIKit
+
+
+/// UIView Class Methods
+///
+extension UIView {
```

Comment:
if this extension serves a specific purpose within WordPressAuthenticator, I'm wondering if it would be better to add a private extension in the same file as `SiteAddressViewController`? Just a suggestion, since we probably already have other `UIView+Helpers` around (e.g. in WordPressUI)

Reply:
🤔 no other extensions have these functions, but I see your point. Authenticator shouldn't be defining them for the rest of the app. I'm going to need these functions in each of the new UIViewControllers. I'll use a protocol instead.

Reply:
Or maybe even just change the file name. If the extension stays internal to the library, consumers should not be able to access it anyway.

Reply:
Extensions default to module access and I didn't make it public. So I had that wrong. This extension wouldn't define the functions outside of Authenticator. I can change the file name. Maybe something like UIView+Auth.swift would be better.